src/main/java/TaskList.java

```diff
@@ -0,0 +1,82 @@
+import java.util.ArrayList;
+
+/**
+ * Class to represent the list of tasks that
+ * are being tracked.
+ */
+public class TaskList {
+
+    private ArrayList<Task> tasks;
+    private int size;
+
+    /**
+     * Constructor.
+     * @param tasks tasks.
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+        this.size = this.tasks.size();
+    }
+
+    /**
+     * Function to add a task into the task list.
```

Comment:
In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.). Instead of starting with "Function...", you could start with the description of what the method is doing (i.e. "Adds a task..."). There should also be an empty line between description and parameter section.

This applies to all methods in this class.